### PR TITLE
Text wrapping

### PR DESCRIPTION
**Pull request by meghanabhange**

### CODE REVIEW

**Comment by meghanabhange** [comment]
Solves for #11

File: _notebooks/nb-stripout.py [hunk 1]
```diff
@@ -10,8 +10,12 @@
 output_stream = io.TextIOWrapper(sys.stdout.buffer, encoding="utf-8")
 
 parser = argparse.ArgumentParser()
-parser.add_argument("-t", "--textconv", action="store_true", help="Print results to output")
-parser.add_argument("-d", "--doc-mode", action="store_true", help="fastai docs nb-specific strip out")
+parser.add_argument(
```
Comment (NirantK) [inline _notebooks/nb-stripout.py]: Please `black *py -l 120`? 

We both use wider displays.

File: requirments.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
 requests
 pydantic
 tqdm
+pysbd
+spacy==0.2.1
```
Comment (NirantK) [inline requirments.txt]: Woah. We don't use that old version of spacy. Use a stable version, something like v2?

File: _notebooks/textbook.py [hunk 3]
```diff
@@ -102,3 +108,29 @@ def get_chapter_pdf_for_book(book: Book) -> List:
                 number=int(file.stem[-2:]),
             )
             self.chapters.append(chp)
+
+    def clean_raw_text(self, disable_pysbd=False):
+        """pySBD (Python Sentence Boundary Disambiguation)
```
Comment (NirantK) [inline _notebooks/textbook.py:109]: Excellent docstring, even if it's a little too verbose :P

File: requirments.txt (hunk 2 above)
Comment (meghanabhange) [reply requirments.txt]: My brain clearly needs more coffee.

File: _notebooks/textbook.py (hunk 3 above)
Comment (meghanabhange) [reply _notebooks/textbook.py:109]: YAY

File: _notebooks/nb-stripout.py (hunk 1 above)
Comment (meghanabhange) [reply _notebooks/nb-stripout.py]: Done 889e390

**Pull request merged**